src/Logging.Legacy/Logging.Legacy.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <PackageId>Microsoft.Orleans.Logging.Legacy</PackageId>
+    <Title>Microsoft Orleans Legacy Logging</Title>
+    <Description>Library of Orleans Legacy Logging.</Description>
```

Comment:
Please have something more descriptive, as this will go into the nuget package metadata. For example:
This package contains some legacy types used for logging to facilitate migration from Orleans 1.x but will be removed in the near future. It is recommended that you use the Microsoft.Extensions.Logging package and its infrastructure and associated providers directly if not upgrading any legacy log consumer.